### PR TITLE
LPS-189491 Replace Collapse by ClayPanel

### PR DESCRIPTION
**Pull request by veroglez**
Dear frontend reviewer,

There are several things that the `ClayPanel` component doesn't have by default, such as a `c-inner` and styles defined in Lexicon. I have opened a question here https://github.com/liferay/clay/issues/5624 for Clay and we have to wait for it to be solved before merging this code.

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FormGeneralPanel.js [hunk 1]
```diff
@@ -95,24 +95,32 @@ export function FormGeneralPanel({item}) {
...
-			<Collapse
-				label={Liferay.Language.get('form-container-options')}
-				open
+		<div className="mb-3 panel-group-sm">
+			<ClayPanel
+				collapsable
+				defaultExpanded={true}
+				displayTitle={Liferay.Language.get('form-container-options')}
+				displayType="unstyled"
+				showCollapseIcon={true}
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FormGeneralPanel.js:104]: `={true}` is not needed we can just left this with defaultExpanend and showCollapsableIcon

Comment (veroglez) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FormGeneralPanel.js]: Yep! my bad! the copy paste is the demon XD

**Comment by ealonso** [comment]
Hey @veroglez,

Can we send this one? or should we wait for the clay ticket?

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:forward

**Comment by alessandromalves** [comment]
ci:test:bundles

**Comment by alessandromalves** [comment]
ci:test:bundle

**Comment by alessandromalves** [comment]
I requested CI to generate a bundle because when doing it locally an error message was appearing when I was up the portal, and it did not appear to be something related to my local environment.

**Comment by alessandromalves** [comment]
The related failures will only need a test fix. See https://github.com/liferay-echo/liferay-portal/pull/13221

**Comment by alessandromalves** [comment]
ci:close

**Pull request closed without merging**